option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message LegionLimitShopBuyRequest
{
 required U64Type legion_id = 1;
 required uint32 index = 2;
 required uint32 merchandise_id = 3;
}
